experimental/apis/data/v0alpha1/errors.go

```diff
@@ -0,0 +1,29 @@
+package v0alpha1
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Represents the state of some operation.
+type Status struct {
```

Comment:
🤣 we get closer and closer to just including [apimachinery](https://github.com/grafana/grafana/tree/main/pkg/apimachinery) and being done with it!